lib/tasks/gettext.rake

```diff
@@ -18,11 +18,13 @@ namespace :gettext do
   end
 
   desc 'Generate POT file'
-  task :pot do
+  task :pot, [:filename] do |_, args|
     begin
-      result = GettextSetup::Pot.generate_new_pot
+      filename = args.filename
+      result = GettextSetup::Pot.generate_new_pot (path: filename)
```

Comment:
The reason i changed the ordering, was because named arguments was failing. You can now see: @smcclellan @eputnam